### PR TITLE
SAK-31473 - Make it easier to configure content-review providers

### PR DESCRIPTION
**Pull request by jonespm**

### CODE REVIEW

**Comment by juanjmerono** [comment]
The PR is for 11.x directly, should this be changed for master?

**Comment by jonespm** [comment]
In master SAK-30207 changed it so you can use a property to change the providers that it knows about, so you don't have to override the values in the bean. I could probably apply a similar change so you could add more services than just the 2 defaulted (Vericite and Turnitin) but this is kind of a stop-gap for 11.0 until that one can get merged in, which could be a few months until 11.1.

So long story short, this is specific to 11 because it's not needed and the change would be different for master.

File: content-review/contentreview-federated/pack/src/webapp/WEB-INF/components.xml [hunk 1]
```diff
@@ -1,24 +1,28 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (juanjmerono) [inline content-review/contentreview-federated/pack/src/webapp/WEB-INF/components.xml]: This line should not be removed.

**Comment by jonespm** [comment]
@juanjmerono Readded the removed xml line

**Pull request merged**